includes/functions.php

```diff
@@ -1260,6 +1260,17 @@ function tsml_import_buffer_set($meetings, $data_source=null) {
 			$meetings[$i]['notes'] .= implode(', ', $unused_types);
 		}
 
+		// If Conference URL, validate; or if phone, force 'ONL' type, else remove 'ONL'
+		$meetings[$i]['types'] = array_values(array_diff($meetings[$i]['types'], array('ONL')));
+		if (!empty($meetings[$i]['conference_url'])) {
+			$url = esc_url_raw($meetings[$i]['conference_url'], array('http', 'https'));
+			if (tsml_conference_provider($url)) {
+				$meetings[$i]['conference_url'] = $url;
+				$meetings[$i]['types'][] = 'ONL';
+			} else $meetings[$i]['conference_url'] = null;
+		}
+		if (!empty($meetings[$i]['conference_phone'])) $meetings[$i]['types'][] = 'ONL'; 
```

Comment:
Might want to make this an `elseif` so that ONL doesn't get added twice

```
} elseif (!empty($meetings[$i]['conference_phone'])) {
  $meetings[$i]['types'][] = 'ONL'; 
}
```

Reply:
The reason I moved away from `elseif` is because if the `conference_url` isn't empty, but doesn't validate, we still want to check to see if the `conference_phone` will add the `ONL` type.